src/App.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import AddView from './Components/AddView/addView';
+import ListView from './Components/ListView/listView';
```

Comment:
React provides for keeping **folders lowercase** (components) and keep **file names of components uppercas**e like the component itself. Also, to avoid merge conflicts, different naming of the same folders should already be adjusted before merging. 
The components folder is already for storing components, there is no need to keep them in separate folders.